devito/builtins.py

```diff
@@ -364,6 +367,29 @@ def buff(i, j):
 
     assign(lhs, rhs, options=options, name=name, **kwargs)
 
+    if pad_halo:
+        pad_fd_halo(function)
+
+
+def pad_fd_halo(function):
```

Comment:
Is `builtins.py` the right place for this?

Reply:
It's just a local function because it's used at two different places, similar to the `nbl` setup function. Not sure where else would put it.

Reply:
yeah, we need to move the builtins to /builtins such that we can also have a utils.py

since it's a private function, the name could start with a _

Reply:
> yeah, we need to move the builtins to /builtins such that we can also have a utils.py

this also appears as an issue on GH. We could do it now

Reply:
Doable

Reply:
Done